src/workerd/io/hibernation-manager.c++

```diff
@@ -134,6 +134,19 @@ void HibernationManagerImpl::hibernateWebSockets(Worker::Lock& lock) {
   });
 }
 
+void HibernationManagerImpl::setEventTimeout(int timeoutMs) {
+  KJ_ASSERT(timeoutMs > 0);
+  eventTimeoutMs = timeoutMs;
+}
+
+void HibernationManagerImpl::unsetEventTimeout() {
+  eventTimeoutMs = kj::none;
+}
```

Comment:
Is a separate `unsetEventTimeout()` method really necessary? This could be accomplished by having `setEventTimeout()` accept a `kj::Maybe`

```cpp
void HibernationManagerImpl::setEventTimeout(kj::Maybe<int> timeoutMs) {
  eventTimeoutMs = timeoutMs.map([](int& val) {
    KJ_ASSERT(val > 0);
    return val;
  });
}
```

Then calling `setEventTimeout(kj::none)` would do the same as `unsetEventTimeout()` here.